### PR TITLE
Communicate provided functionality, intended uses, and relationship to other gems. Fixes #53.

### LINKED ISSUE #53: Update README and Guidelines for Contributing to reflect current developments
The [README](https://github.com/projecthydra-labs/hydra-works/blob/master/README.md) is incomplete and out of date.  It needs to 
- [x] Declare what functionality is provided by the gem
- [x] Declare the intended uses of the gem
- [x] Declare the relationship between this code and Sufia, sufia-models, Worthwhile and Curate
- [x] Provide Instructions for Installation/Use 
- [x] Provide links to background documentation (ie. Application Profile) with context explaining the relationship between that documentation and this code

The  [Guidelines for Contributing](https://github.com/projecthydra-labs/hydra-works/blob/master/CONTRIBUTING.md) still say that we are soliciting Use Cases.  They don't say anything about contributing code or submitting issues other than Use Cases.  They should
- [x] Provide guidelines for contributing code
- [x] Provide guildelines for submitting issues and pull requests
Is there a template for this somewhere in the Hydra space?  If not, should we create one and apply it to all of the repositories?

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,31 +38,11 @@ Or install it yourself as:
...
-
-A `Hydra::PCDM::File` is a NonRDFSource &emdash; a bitstream.  You can use this to store content. A PCDM::File is contained by a PCDM::Object. A `File` has some attached technical metadata, but no descriptive metadata.  A `Hydra::PCDM::Object` contains files and other objects and may have descriptive metadata.  A `Hydra::PCDM::Collection` can contain other `Collection`s or `Object`s but not `File`s.  A `Collection` also may have descriptive metadata.
-
-Typically usage involves extending the behavior provided by this gem. In your application you can write something like this:
+Usage involves extending the behavior provided by this gem. In your application, you can create Hydra::Works-based models like so:
 
 ```ruby
-
 class Collection < ActiveFedora::Base
   include Hydra::PCDM::CollectionBehavior
```
Comment (flyingzumwalt) [inline README.md]: Remove this line. The Works::CollectionBehavior includes PCDM::Collection behavior for you.

File: README.md [hunk 2]
```diff
@@ -70,7 +58,6 @@ class BookFiles < ActiveFedora::Base
   include Hydra::Works::GenericFileBehavior
```
Comment (flyingzumwalt) [inline README.md]: This should not be here.  Collections are not Files.

File: README.md [hunk 3]
```diff
@@ -2,27 +2,35 @@
...
+Hydra::Works, which was inspired by Sufia, Worthwhile, and Curate, will provide interoperable models for use by a number of Hydra tools, including the aforementioned gems and more. Hydra::Works adoption need not be limited to the institutional/data repository use case; that is merely the starting point.
 
 ## Installation
 
 Add these lines to your application's Gemfile:
 
 ```ruby
-  gem 'active-fedora', github: 'projecthydra/active_fedora' # hydra-pcdm requires an unreleased version of ActiveFedora
+  # hydra-pcdm requires an unreleased version of ActiveFedora
+  gem 'active-fedora', github: 'projecthydra/active_fedora'
```
Comment (flyingzumwalt) [inline README.md]: You only need hydra-works in your Gemfile.  It includes the others in its dependencies.

File: README.md [hunk 4]
```diff
@@ -2,27 +2,35 @@
...
 
+ * **GenericFile**: a *pcdm:Object* that encapsulates one or more directly related *pcdm:File*s, such as a PDF document, its derivatives, and extracted full-text
+ * **GenericWork**: a *pcdm:Object* that holds zero or more **GenericFile**s and zero or more **GenericWork**s
+ * **Collection**: a *pcdm:Collection* that indirectly contains zero or more **GenericWork**s and zero or more **Collection**s
+
+View [a diagram of the domain model](https://docs.google.com/drawings/d/1-NkkRPpGpZGoTimEpYTaGM1uUPRaT0SamuWDITvtG_8/edit).
+
+Hydra::Works, which was inspired by Sufia, Worthwhile, and Curate, will provide interoperable models for use by a number of Hydra tools, including the aforementioned gems and more. Hydra::Works adoption need not be limited to the institutional/data repository use case; that is merely the starting point.
```
Comment (flyingzumwalt) [inline README.md]: This needs to be clearer -- Hydra::Works is not a consolidation of the use cases for Sufia, Worthwhile, and Curate -- that would be CurationEngine.  Hydra::Works provides a core implementation of PCDM-based multi-file "works" that can be used by applications like Sufia, Worthwhile, and Curate.

File: README.md [hunk 5]
```diff
@@ -2,27 +2,35 @@
 [![Build Status](https://travis-ci.org/projecthydra-labs/hydra-works.svg?branch=master)](https://travis-ci.org/projecthydra-labs/hydra-works)
 [![Coverage Status](https://coveralls.io/repos/projecthydra-labs/hydra-works/badge.svg?branch=master)](https://coveralls.io/r/projecthydra-labs/hydra-works?branch=master)
 
-Hydra implementation of a works model based on the [Portland Common Data Model](https://github.com/projecthydra-labs/hydra-pcdm) (PCDM).
+The Hydra::Works gem provides a set of [Portland Common Data Model](https://wiki.duraspace.org/display/FF/Portland+Common+Data+Model)-compliant models and associated behaviors around the broad concept of multi-file "works," the need for which was expressed by a variety of [community use cases](https://github.com/projecthydra-labs/hydra-works/tree/master/use-cases). The Hydra::Works domain model includes:
```
Comment (flyingzumwalt) [inline README.md]: typo: `multi-file "works",` not `multi-file "works,"`

Comment (mjgiarlo) [inline README.md]: Grammatically speaking, commas and periods always go inside quotation marks in American English.  If you think this is inconsistent with the rest of the README, or with common practice in our documentation, I'm happy to make the change.

File: README.md (hunk 3 above)
Comment (mjgiarlo) [inline README.md]: This stuff came in via what's in master. @elrayle ?

File: README.md (hunk 1 above)
Comment (mjgiarlo) [inline README.md]: @elrayle did you find this not to be true?

File: README.md (hunk 2 above)
Comment (mjgiarlo) [inline README.md]: Not sure I follow. Where do you see references to collections in `BookFiles`?

File: README.md (hunk 4 above)
Comment (mjgiarlo) [inline README.md]: I'm inclined to strike the whole paragraph. This seems like one of those instances where saying less is wiser and claerer than saying more.

**Comment by flyingzumwalt** [comment]
:+1: any objections to merging?

**Comment by mjgiarlo** [comment]
:japanese_ogre: @flyingzumwalt

**Comment by flyingzumwalt** [comment]
_@flyingzumwalt considers merging even though travis is failing_ :japanese_goblin: What could go wrong?

File: README.md [hunk 6]
```diff
@@ -2,75 +2,45 @@
...
+ * **Collection**: a *pcdm:Collection* that indirectly contains zero or more **GenericWork**s and zero or more **Collection**s
+
+View [a diagram of the domain model](https://docs.google.com/drawings/d/1-NkkRPpGpZGoTimEpYTaGM1uUPRaT0SamuWDITvtG_8/edit).
 
 ## Installation
 
 Add these lines to your application's Gemfile:
 
-```ruby
-  gem 'active-fedora', github: 'projecthydra/active_fedora' # hydra-pcdm requires an unreleased version of ActiveFedora
```
Comment (jcoyne) [inline README.md]: These lines are important until we have cut new gems

Comment (mjgiarlo) [inline README.md]: @flyingzumwalt suggested we didn't need them. I asked @elrayle in channel and she didn't say otherwise. Sry!  I'll edit them back in.

Comment (flyingzumwalt) [inline README.md]: Doh! My bad.

Comment (mjgiarlo) [inline README.md]: np!  :japanese_goblin:

**Pull request merged**